runtime/onert/core/src/compiler/HEScheduler.cc

```diff
@@ -204,7 +204,12 @@ void HEScheduler::scheduleBranch(const ir::OperationIndex &index,
       return;
     }
     const auto &only_out_operand = _graph->operands().at(*node.getOutputs().begin());
-    loc_index = only_out_operand.getUses().list().front();
+    // One of the last nodes
+    if (only_out_operand.getUses().size() == 0)
+    {
+      return;
+    }
+    loc_index = *only_out_operand.getUses().begin();
```

Comment:
@ragmani it seems it does early exit that doesn't exist. I think it is beyond this PR scope (which already has several changes at one time).

Reply:
Why does it early exit? If `only_out_operand` does not have `uses`, there is no reason to call codes below

Reply:
@ragmani Thank you for explanation. I am saying it seems beyond scope of this PR.

Reply:
Right. But this bug makes segmentation fault in this PR. I hope this bug is fixed with this PR.